extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -728,14 +780,14 @@ This will either:
 `commercetools-adyen-integration` supports multi-tenancy to serve multiple Adyen merchant accounts/commercetools projects
 with one application instance. This architectural style leverages sharing and scalability to provide cost-efficient hosting.
 
-In order for `commercetools-adyen-integration` to know which project it should communicate with, this information must be provided. Payment object must contain the following 2 custom fields:
+In order for `commercetools-adyen-integration` to know which project it should communicate with, this information must be provided. The payment object must contain the following 2 custom fields:
```

Comment:
Maybe we can skip it as it is mandatory for single tenants too and we mention it on payment creation already?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/570/commits/e4b122c4195da1eeb025d07d6785f41603281b7f